templates/launch.html

```diff
@@ -118,10 +118,32 @@ <h3>Runner Data</h3>
                             defaultValue = metadataField['default']
                         }
 
-                        var metadataFieldHtml = "<div class=\"field-container\">" +
-                                "<label for=\"" + metadataField['name'] + "\">" + metadataField['name'] + "</label>" +
-                                "<input id=\"" + metadataField['name'] + "\" name=\"" + metadataField['name'] + "\" type=\"text\" value=\"" + defaultValue + "\" class=\"" + metadataField['name'] + "\">" +
-                                "</div>"
+                        var metadataFieldHtml = ""
+
+                        if (metadataField['validator'] == "boolean") {
+
+                            metadataFieldHtml = "<div class=\"field-container\">" +
+                                    "<label for=\"" + metadataField['name'] + "\">" + metadataField['name'] + "</label>" +
+                                    "<input id=\"" + metadataField['name'] + "\" type=\"checkbox\" name=\"" + metadataField['name'] + "\" class=\"qa-" + metadataField['name'] + "\" value=\"true\">" +
+                                    "</div>"
+
+                        } else if (metadataField['validator'] == "uuid") {
+
+                            metadataFieldHtml = "<div class=\"field-container\">" +
+                                    "<label for=\"" + metadataField['name'] + "\">" + metadataField['name'] + "</label>" +
+                                    "<span>" +
+                                    "<input id=\"" + metadataField['name'] + "\" name=\"" + metadataField['name'] + "\" type=\"text\" value=\"" + get_uuid() + "\" class=\"" + metadataField['name'] + "\">" +
+                                    "<img onclick=\"uuid('" + metadataField['name'] + "')\" src=\"data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/PjwhRE9DVFlQRSBzdmcgIFBVQkxJQyAnLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4nICAnaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkJz48c3ZnIGhlaWdodD0iNTEycHgiIGlkPSJMYXllcl8xIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA1MTIgNTEyOyIgdmVyc2lvbj0iMS4xIiB2aWV3Qm94PSIwIDAgNTEyIDUxMiIgd2lkdGg9IjUxMnB4IiB4bWw6c3BhY2U9InByZXNlcnZlIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIj48Zz48cGF0aCBkPSJNMjU2LDM4NC4xYy03MC43LDAtMTI4LTU3LjMtMTI4LTEyOC4xYzAtNzAuOCw1Ny4zLTEyOC4xLDEyOC0xMjguMVY4NGw5Niw2NGwtOTYsNTUuN3YtNTUuOCAgIGMtNTkuNiwwLTEwOC4xLDQ4LjUtMTA4LjEsMTA4LjFjMCw1OS42LDQ4LjUsMTA4LjEsMTA4LjEsMTA4LjFTMzY0LjEsMzE2LDM2NC4xLDI1NkgzODRDMzg0LDMyNywzMjYuNywzODQuMSwyNTYsMzg0LjF6Ii8+PC9nPjwvc3ZnPg==\">" +
```

Comment:
This code assumes that there won't be spaces in the name, else it wont be a valid html `id` field. Choose your destiny. Pick one of the following:

1. The schema validator doesn't actually validate the metadata names, other than that they're strings at the moment. We should update this to regex match `/[a-zA-Z0-9\.-_]+/` so that the name can be used safely.
2. Base64 encode the metadataField['name'] when using it as a class name, or you could grab a method to camelCase if you wanted, but due to the fact that there could be any valid string as a name at the moment, base64 makes less assumptions.

Reply:
I think require no spaces is probably best? given that in runner when we define it in the schema, i dont think that will work if the key has spaces?
eq, when we do `"metadata.something"` it wont work as `"metadata.some thing"`?

Reply:
Yeah, that makes sense, it's safer overall! I'd propose the regex I put in is probably permissive enough, though for the scenario you described we need to not allow `.` too.

Reply:
PR raised https://github.com/ONSdigital/eq-schema-validator/pull/74